elasticapm/traces.py

```diff
@@ -102,7 +112,8 @@ def to_dict(self):
             'context': self._context,
             'spans': [
```

Comment:
I think it's worth considering removing the `spans` key entirely to clearly indicate this is an unsampled transaction.